.travis.yml

```diff
@@ -70,20 +71,24 @@ jobs:
   - stage: publish amd64
     if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
     script:
-    - travis_wait ${TRAVIS_TIMEOUT} .travis/publish.sh amd64
+    - .travis/publish.sh amd64
```

Comment:
Who do we not need `TRAVIS_TIMEOUT` anymore?

Reply:
This was required only for the s390x and ppc64le nginx images. Right now I am building that with an AWS VM